### PR TITLE
Update SecretDecks® Newsletter to Occultish.md

### PR DESCRIPTION
**Pull request by evolayak**
did a very minor proofread (not sure if this went out yet, tho!)

### CODE REVIEW

File: Newsletters/SecretDecks® Newsletter to Occultish.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
 ## Calling all creators! 
 SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
+
 ## Are you an artist?  
-Build a new global fanbase, make a living off your digital art without any middlemen and through the medium of your choice, as well as own and manage your copyrights!
-## What if you're not an aritst? 
+Build a new global fanbase and make a living off your digital art without any middlemen and through the medium of your choice, as well as own and manage your copyrights!
```
Comment (wlstylr) [inline Newsletters/SecretDecks® Newsletter to Occultish.md:5]: looks good

File: Newsletters/SecretDecks® Newsletter to Occultish.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+
+## What if you're not an artist? 
 Join us anyway and discover how we can help each other build an ecosystem of mutual abundance. 
 There's enough to do for everyone, as we're building the future of decentralised collaboration and autonomus organisation. 
 
 ## Tell me more.
-We want to give you the chance to showcase and sell your unique Tarot recreation as digital collectibles in a fresh, innovative way. It’s a method we believe not only has huge momentum now - it also builds on a technology that is rocketing into the future while allowing art and its creators more autonomy. 
+We want to give you the chance to showcase and sell your unique Tarot recreations as digital collectibles in a fresh, innovative way. It’s a method we believe not only has huge momentum now, but also builds on a technology that is rocketing into the future while allowing art and its creators more autonomy.
```
Comment (wlstylr) [inline Newsletters/SecretDecks® Newsletter to Occultish.md:12]: Yes - seems better

File: Newsletters/SecretDecks® Newsletter to Occultish.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
 
-How does an NFT built on the blockchain differ from simply selling it from a website on the internet? There is no website where you can sell unique digital art pieces without the need to set up a payment gateway. Additionally, selling art as an NFT enables lifelong royalties. Each time a digital artwork is resold, the artist receives a royalty. 
+How does an NFT built on the blockchain differ from simply selling it on a website on the internet? There is no website where you can sell unique digital art pieces without the need to set up a payment gateway. Additionally, selling art as an NFT enables lifelong royalties. Each time a digital artwork is resold, the artist receives a royalty.
```
Comment (wlstylr) [inline Newsletters/SecretDecks® Newsletter to Occultish.md:19]: I feel like a website in the ecommerce is synonymous with "market" or "store."  I would usually say things like "I buy things from a store," or "from a marketplace,"  not "on a website."

Alternatively art is typically displayed "on a website." So I'm not 100% sure what works better.

File: Newsletters/SecretDecks® Newsletter to Occultish.md [hunk 4]
```diff
@@ -32,13 +34,14 @@ An organisational layer (network) of groups of creators clustered around an idea
 ## How do they come together?
 Think digital collectibles in the form of Tarot cards, conjured up by creators across the globe and distributed and utlised within an ecosystem of SecretDecks® apps that work together to fuel demand for the collectible cards and the experiences they entail.
 
-In order to generate interest and value, as well as motivate participants and test the ACO concept in a real world environment, it is required to have some value-added mechanism. Thus a small percentage (appr. 0.5%-1%) of the collectible sales will flow into the ACO network. 
+In order to generate interest and value, as well as motivate participants and test the ACO concept in a real world environment, it is required to have some value-added mechanism. Thus, a small percentage (appr. 0.5%-1%) of the collectible sales will flow into the ACO network.
```
Comment (wlstylr) [inline Newsletters/SecretDecks® Newsletter to Occultish.md:37]: Looks good

**Review by wlstylr — COMMENTED** [review]
Thank you Heather!  This mostly all looks great, but I wasn't sure about the change from "from" to "on" when describing the action of selling through a website.

File: Newsletters/SecretDecks® Newsletter to Occultish.md (hunk 1 above)
Comment (evolayak) [reply Newsletters/SecretDecks® Newsletter to Occultish.md:5]: Ah yes. I used 'on" because it's sold on a platform, but "from" also works!

**Pull request merged**